index.html

```diff
@@ -53,13 +50,14 @@
 	      "PR" : "https://www.w3.org/TR/wai-aria-1.1/",
 	      "REC": "https://www.w3.org/TR/wai-aria/"
 	  },
-
-	  preProcess: [ linkCrossReferences ]
+	    preProcess:[linkCrossReferences],
+		postProcess:[],
+		definitionMap:[],
+		xref: ["core-aam", "accname", "wai-aria"]
```

Comment:
can you explain why this is needed in this PR?

Reply:
- linkCrossReferences deals with links between different ARIA specs (rref, pref etc.) and resolves them correctly
- xref deals will look in those other specs for references and allows the terms to be referenced correctly.